staging/src/k8s.io/apiserver/pkg/registry/generic/registry/store.go

```diff
@@ -413,7 +413,7 @@ func (e *Store) WaitForInitialized(ctx genericapirequest.Context, obj runtime.Ob
 
 // shouldDeleteDuringUpdate checks if a Update is removing all the object's
 // finalizers. If so, it further checks if the object's
-// DeletionGracePeriodSeconds is 0. If so, it returns true.
```

Comment:
this merged upstream, drop this commit completely (cosmetic changes only)

Reply:
Dropped